canonical-web-and-design/snapcraft.io
# Upgrade to vanilla v2.7.1

## Done

- Upgrade to vanilla v2.7.1
- Replace `p-heading--one, --two, ...` with `p-heading--1, --2, ...`) 
- Remove navigation overrides

## QA

- Pull the branch
- Run the site using the command `./run`
- View the site locally in your web browser at: http://0.0.0.0:8004/
- Run through the following [QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html)
- Check that all looks right
